frame/nomination-pools/src/lib.rs

```diff
@@ -2311,7 +2601,28 @@ impl<T: Config> Pallet<T> {
 		member.last_recorded_reward_counter = current_reward_counter;
 		reward_pool.register_claimed_reward(pending_rewards);
 
-		// Transfer payout to the member.
+		// Gets the commission percentage and payee to be paid if commission has
+		// been set. Otherwise, a zero commission and payee of `None` is returned.
+		let commission_current = &bonded_pool.commission.get_commission_and_payee(&pending_rewards);
+
+		if let Some((pool_commission, payee)) = commission_current {
+			// Deduct any outstanding commission from the reward being claimed.
+			pending_rewards = pending_rewards.saturating_sub(*pool_commission);
+
+			// If a non-zero commission has been applied to the pool, deduct the share from
+			// `reward_pool` and send the amount to the commission `payee`.
+			// Defensive: The commission payee is also checked for existence.
```

Comment:
where is this checked?

Reply:
This used to be a double `if let` where the latter one checked the payee. This last comment is no longer valid as the whole `current` tuple is checked together, I will remove.